glue_jupyter/bqplot/common/tools.py

```diff
@@ -348,6 +360,21 @@ def activate(self):
         super().activate()
 
 
+@viewer_tool
+class BqplotTrueCircleMode(BqplotCircleMode):
+
+    tool_id = 'bqplot:truecircle'
+    tool_tip = 'Define a strictly circular region of interest'
+
+    def __init__(self, viewer, roi=None, finalize_callback=None, **kwargs):
+
+        super().__init__(viewer, **kwargs)
+
+        if kwargs.pop('strict_circle', True) is False:
```

Comment:
Yeah, I agree this is redundant. What do you think, @astrofrog ?

Reply:
Yes agree